src/documentation/app/AppContainer.jsx

```diff
@@ -75,7 +54,7 @@ export default class AppContainer extends React.Component {
   renderMarkdownComponent = () => <markdownToJsx>{UIComponent}</markdownToJsx>
 
   render() {
-    const { route, location } = this.state;
+    const { match: { params: { route, location, view } } } = this.props;
```

Comment:
Ah, thanks so much for fixing this @hamstu 🙇 !

Reply:
Just a little bit of healthy destructuring! 😂